chocolatey/lektor.nuspec

```diff
@@ -0,0 +1,34 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>lektor</id>
+    <version>3.2.0</version>
+    <packageSourceUrl>https://github.com/lektor/lektor</packageSourceUrl>
+    <owners>Nimish Bongale</owners>
+
+    <title>lektor</title>
+    <authors>lektor</authors>
+    <projectUrl>https://www.getlektor.com/</projectUrl>
+    <!--<iconUrl>http://cdn.rawgit.com/__REPLACE_YOUR_REPO__/master/icons/lektor.png</iconUrl>-->
+    <!-- <copyright>Year Software Vendor</copyright> -->
+    <licenseUrl>https://github.com/lektor/lektor/blob/master/LICENSE</licenseUrl>
```

Comment:
https://www.getlektor.com/license/ <-- this is probably the more user friendly URL.

Or does this actually include the license itself in the installer? If so you should probably use the raw URL: https://raw.githubusercontent.com/lektor/lektor/master/LICENSE

Reply:
Ahhh alright. The LICENSE in the installer is for a different purpose I think. Will check.